conf/uppmax.config

```diff
@@ -19,22 +19,22 @@ process {
   maxRetries = 3
   maxErrors = '-1'
 
-    $FusionCatcher{
+  $fusioncatcher{
     module = ['bioinfo-tools', 'FusionCatcher/0.99.5a']
     cpus = { 16 * task.attempt }
     memory = { 32.GB * task.attempt }
-    }
-    $STAR_fusion{
+  }
+  $star_fusion{
     module = ['bioinfo-tools', 'star-fusion']
     cpus = { 4 * task.attempt }
     memory = { 32.GB * task.attempt }
-    }
+  }
+}
 
-    params {
-        references{
-            fusioncatcer_data = '/sw/apps/bioinfo/FusionCatcher/0.99.5a/milou/data/ensembl_v83/'
-            star_fusion_reference  = '/proj/b2013064/private/nobackup/rickard/STAR_fusion_refrence/GRCh37_gencode_v19_CTAT_lib_July272016_prebuilt'
-            fusioncatcher_data_dir = '/sw/apps/bioinfo/FusionCatcher/0.99.5a/milou/data/current'
-            }
-    }
+params {
+  references{
```

Comment:
Note yet, or the `fusioncatcher_data_dir` is supposed to be used in the `fusioncatcher` process. Maybe I mad a typo? I tend to do that